lib/yabeda/activejob.rb

```diff
@@ -96,7 +96,7 @@ def self.job_latency(event)
       enqueue_time = event.payload[:job].enqueued_at
       return nil unless enqueue_time.present?
 
-      enqueue_time = Time.parse(enqueue_time).utc
+      enqueue_time = Time.parse(enqueue_time).utc unless enqueue_time.is_a?(Time)
```

Comment:
Interesting something about this, change is breaking existing specs the specs failing look to be around latency tests I haven't touched this code in a while 😅 let me know if you need help debugging and i can take a look sometime next week